legal-api/requirements/bcregistry-libraries.txt

```diff
@@ -1 +1 @@
-git+https://github.com/bcgov/business-schemas.git@2.5.12#egg=registry_schemas
+git+https://github.com/bcgov/business-schemas.git#egg=registry_schemas
```

Comment:
Are you sure that you don't want to control the version of the schema you use?

Reply:
yeah we will want this controlled I think

Reply:
If you want to bring in the schema changes then you can do a gh release in the schema repo for .13 and then point to version .13 here

Reply:
oh we don't want the most recent? Ok will update

Reply:
Currently, you probably want the most recent (13).
But we may want the API to use a set version of the schema (eg, 12), for example if the API is not yet ready to handle the latest schema changes. Or maybe for a hotfix.

The UIs do the same thing for the sbc-common-components. It allows the UI to control when it upgrades to the latest library (instead of automatically picking up the latest changes, which may be breaking!).

Reply:
The version doesn't need to be here. It does need to be in the requirements.txt file.

Reply:
should be fixed now

Reply:
Why does the diff still show that you deleted "@2.5.12" ?

Is this is a generated file, so it doesn't matter?

Reply:
well, if the version doesn't need to be here I figured you probably want the most recent version if rebuilding the requirements locally. I had it @2.5.13 but if the version isn't needed in this file I'll drop it as 2.5.12 would be incorrect